instructors.md

```diff
@@ -529,36 +551,35 @@ Solutions to exercises:
 >         assert result[-1] >= result[0]
 >     return result
 > ~~~
-> ~~~ {.output}
-> The first assertion checks that the input sequence `values` is not empty.
->   An empty sequence such as `[]` will make it fail.
-> The second assertion checks that the first value in the list is positive.
->   Input such as `[-1,0,2,3]` will make it fail.
-> The third assertion checks that the running total always increases.
->   Input such as `[0,1,3,-5,4]` will make it fail.
-> ~~~
+>
```

Comment:
I think we were using .output to indicate that this was the solution. Do you think we need some kind of visual cue to separate the question from the answer?